packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/CheckoutConfirm/index.tsx

```diff
@@ -33,48 +33,18 @@ class CheckoutConfirm extends PureComponent<Props> {
     )
     this.props.sendActions.getLockRule()
     if (!Remote.Success.is(this.props.data)) {
-      this.props.simpleBuyActions.fetchSDDEligible()
-      this.props.simpleBuyActions.fetchSDDVerified()
-      this.props.simpleBuyActions.fetchSBCards()
```

Comment:
why are we removing fetching cards?

Reply:
cards were used only for sdd part since this PR removes all of that we can remove loading cards at this point

Reply:
Ah, I thought we were fetching cards on checkout confirm just in case any of them got blocked or voided before the user confirmed their buy. If that's not the case then you can probably remove `this.props.simpleBuyActions.fetchBankTransferAccounts()` too

Reply:
yup I removed this one easily